### ISSUE TITLE
Using glossaries package for lists of symbols, abbreviations, and etc.

### ISSUE THREAD

**Issue by piyanatk** [GarenSidonius/ASU-Dissertation-Template#16]
I am trying to use `glossaries` package to create list of symbols and abbreviations. 

This sample codes work and produce a [PDF output](https://github.com/GarenSidonius/ASU-Dissertation-Template/files/3056574/sample_glossaries.pdf) with correct ToC and hyperref links.

```latex
\documentclass{memoir}

\title{A La\TeX Article}
\author{Your name here}
\date{\today}

\usepackage{hyperref}
\usepackage[acronym,toc]{glossaries}

% abbreviations:
\newacronym{ny}{NY}{New York}
\newacronym{la}{LA}{Los Angeles}
\newacronym{un}{UN}{United Nations}

% nomenclature:
\newglossaryentry{angelsperarea}{
  name = $a$ ,
  description = The number of angels per unit area,
}
\newglossaryentry{numofangels}{
  name = $N$ ,
  description = The number of angels per needle point
}
\newglossaryentry{areaofneedle}{
  name = $A$ ,
  description = The area of the needle point
}

\makeglossaries

\begin{document}
    \frontmatter

    \maketitle

    \clearpage
    \tableofcontents

    \printglossary[type=\acronymtype,title=Abbreviations]

    \printglossary[title=Symbols]

    \mainmatter

    \chapter{Introduction}
    \gls{ny}, \gls{la} and \gls{un} are abbreviations whereas
    \gls{angelsperarea} is part of the symbols

    \chapter{Next}
    \gls{angelsperarea}, \gls{numofangels} and \gls{areaofneedle} are symbols
\end{document}
```

When similar codes are used in the template, the [output](https://github.com/GarenSidonius/ASU-Dissertation-Template/files/3056603/dissertation_template_latex_sample.pdf) has a correct ToC, but the hyperref links from the Abbreviations or Symbols to their apperances in the main text do not account for pages with roman page numbers - i.e., they link to the actual PDF pages. However, contents in the main text do correctly link back to the lists. Do you have a clue what in the template is conflicting this use?
```latex
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Glossaries
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Undo the wrong changes made by mathspec
% See https://tex.stackexchange.com/questions/85696/
\makeatletter
\let\RequirePackage\original@RequirePackage
\let\usepackage\RequirePackage
\makeatother

\usepackage[acronym,toc]{glossaries}

% abbreviations:
\newacronym{ny}{NY}{New York}
\newacronym{la}{LA}{Los Angeles}
\newacronym{un}{UN}{United Nations}

% nomenclature:
\newglossaryentry{angelsperarea}{
  name = $a$ ,
  description = The number of angels per unit area,
}
\newglossaryentry{numofangels}{
  name = $N$ ,
  description = The number of angels per needle point
}
\newglossaryentry{areaofneedle}{
  name = $A$ ,
  description = The area of the needle point
}

\makeglossaries

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Copyright page
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
```latex
\chapter{This is a chapter-level heading}

\lipsum[1]

\section{Links to Glossaries}
    \gls{ny}, \gls{la} and \gls{un} are abbreviations whereas
    \gls{angelsperarea}, \gls{numofangels} and \gls{areaofneedle} are symbols
```

**Comment by piyanatk** [comment]
The `memoir` class does provide means for a glossary, using the `\glossary` command and displaying it with `\printglossary`. However, there's no` \gls` or related command that does display the glossary item inline, i.e. outside of the glossary itself, as in the `glossaries` package. This function is very helpful for a long scientific work because it allows you to define all of your abbreviations, symbols and terms once and insert them in the main text.

My point in the post above is that `glossaries` works correctly with the plain `memoir` class. Therefore, some tweaking in this template must have conflict with how `glossaries` do hyperref links, and I just wondering if you have a clue of what might cause the issue.

**Comment by piyanatk** [comment]
I figured out the issue. It is caused by a conflict in Arabic numeral page numbering. The title and copyright pages are printed before the frontmatter, so they default to Arabic numeral page numbering even though the page numbers are suppressed. When glossary terms are displayed in the first two pages of the mainmatter, which also have Arabic numeral page numbering, they will be linked to the first occurrences of the corresponding Arabic numeral pages, which are the title and copyright pages. If glossary terms are inserted in page 3+ of the mainmatter, there will be no issue.

I think this conflict should be fixed regardless of the use of the `glossaries` package. The easiest way to do this is to force alphabetic page numbering for the title and copyright pages before the frontmatter.
```latex
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Document
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

\pagenumbering{alph}   % Force alphabetic page numbering for title and copyright pages

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Title page
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\titlepageASU

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Copyright page
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\copyrightpageASU                       %~If you don't want to have a copyright page,
                                        %   comment out this line

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Front matter
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapterstyle{ASU}
\pagestyle{ASU}
\frontmatter  % Page numbering is automatically set to roman
```

**Comment by GarenSidonius** [comment]
Fantastic, I've updated the template with this fix and supporting documentation: 61150d492c959a1ddec9361177d3e1b3ce167224
